### PR TITLE
add redis tests

### LINKED ISSUE #415: [testing] redis test suite
add test suite just to make sure redis is working

### PR DESCRIPTION
**Pull request by mchangrh**
closes #415

### CODE REVIEW

**Comment by mchangrh** [comment]
not sure if the CI should be renamed but it works 🥳

**Comment by mchangrh** [comment]
there's one test that keeps failing, will look into it and keep it drafted until then

File: test/cases/ratings/getRating.ts [hunk 1]
```diff
@@ -80,7 +83,7 @@ describe("getRating", () => {
                     count: 10,
                     hash: videoOneIDHash,
                 }];
-                assert.ok(partialDeepEquals(res.data, expected));
+                assert.ok(arrayPartialDeepEquals(res.data, expected));
```
Comment (ajayyy) [inline test/cases/ratings/getRating.ts:86]: Is this due to changing order?

Comment (mchangrh) [reply test/cases/ratings/getRating.ts:86]: yep, the order would randomly change. arrayPartialDeepEquals is a lot heavier so I only want to use it when partialDeepEquals doesn't return consistent results

Comment (mchangrh) [reply test/cases/ratings/getRating.ts:86]: for some reason, this was the **only** test that would consistently fail for any subsequent tests on redis

**Comment by mchangrh** [comment]
will fix lint issues

**Comment by ajayyy** [comment]
oops, in another pr

**Comment by ajayyy** [comment]
wanted this for #431

**Pull request merged**